Banyan/src/sessions.jl

```diff
@@ -342,4 +345,35 @@ function with_session(f::Function; kwargs...)
             end_session(j)
         end
     end
+end
+
+
+function run_session(;
+    cluster_name::Union{String,Nothing} = nothing,
+    nworkers::Union{Integer,Nothing} = 16,
+    release_resources_after::Union{Integer,Nothing} = 20,
+    print_logs::Union{Bool,Nothing} = false,
+    store_logs_in_s3::Union{Bool,Nothing} = true,
+    store_logs_on_cluster::Union{Bool,Nothing} = false,
+    sample_rate::Union{Integer,Nothing} = nworkers,
+    session_name::Union{String,Nothing} = nothing,
+    files::Union{Vector,Nothing} = [],
+    code_files::Union{Vector,Nothing} = [],
+    pf_dispatch_table::Union{String,Nothing} = nothing,
+    using_modules::Union{Vector,Nothing} = [],
+    url::Union{String,Nothing} = nothing,
+    branch::Union{String,Nothing} = nothing,
+    directory::Union{String,Nothing} = nothing,
+    dev_paths::Union{Vector,Nothing} = [],
+    force_clone::Union{Bool,Nothing} = false,
+    force_pull::Union{Bool,Nothing} = false,
+    force_install::Union{Bool,Nothing} = false,
+    estimate_available_memory::Union{Bool,Nothing} = true,
+    email_when_ready::Union{Bool,Nothing}=nothing,
+    kwargs...,)::SessionId
```

Comment:
Maybe @calebwin knows, but I"m not sure it's needed to list out all the params like this.
I'm wondering whether run_session can just accept kwargs, and then pass that to start_session?

Reply:
That's probably better but there are some arguments that `start_session` accepts that `run_session` doesn't like `force_update_files`. I think for now this is fine.